src/app/app-nav-bar/app-nav-bar.tsx

```diff
@@ -25,123 +27,154 @@ interface Props {
   onChangeLanguage: (language: string) => void;
 }
 
-const getMenuItems: any = (isNap: boolean) =>
-  isNap
-    ? [
-        {
-          label: localization.menu.aboutNap,
-          url: PATHNAME_ABOUT_NAP,
-          hrefLink: true
-        },
-        {
-          label: localization.menu.aboutRegistration,
-          url: PATHNAME_ABOUT_REGISTRATION
-        },
-        {
-          label: localization.menu.organizations,
-          url: PATHNAME_ORGANIZATIONS
-        },
-        {
-          label: localization.menu.reports,
-          url: PATHNAME_REPORTS
-        }
-      ]
-    : [
-        {
-          label: localization.menu.about,
-          url: PATHNAME_ABOUT
-        },
-        {
-          label: localization.menu.organizations,
-          url: PATHNAME_ORGANIZATIONS
-        },
-        {
-          label: localization.menu.reports,
-          url: PATHNAME_REPORTS
-        },
-        {
-          label: localization.menu.publishing,
-          url: PATHNAME_PUBLISHING,
-          hrefLink: true,
-          externalIcon: true
-        }
-      ];
-
-const MenuItems: FC = () =>
-  getMenuItems(isTransportportal).map(
-    ({ label, url, hrefLink = false, externalIcon = false }: any) => (
-      <li key={label}>
-        {hrefLink ? (
-          <SC.Link href={url} {...(externalIcon && { external: true })}>
-            {label}
-          </SC.Link>
-        ) : (
-          <SC.Link
-            as={RouteLink}
-            to={url}
-            {...(externalIcon && { external: true })}
-          >
-            {label}
-          </SC.Link>
-        )}
+const menuItems = () =>
```

Comment:
Kan du refaktorere mer? F.eks. ha menypunkter for FDK og Transportportal i hver sin meny-liste slik at man ikke må vedlikeholde menypunktene to steder i markup, det var noe av tanken med 'getMenuItems'. Hvis vi kan ha en UI-komponent som er uavhengig av de faktiske menypunktene, det kommer som props.